.github/workflows/codeql-analysis.yml

```diff
@@ -34,8 +37,11 @@ jobs:
         languages: ${{ matrix.language }}
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
-    - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+    #- name: Autobuild
+    #  uses: github/codeql-action/autobuild@v1
+      
```

Comment:
Commented out sections aren't needed. If this info is needed in the future, it can be found in git's history.